tasks/container/cockpit-tasks

```diff
@@ -41,7 +29,7 @@ for i in $(seq 1 30); do
     cd "$BOTS_DIR"
 
     # run-queue fails on empty queues; don't poll too often
-    timeout 12h ./run-queue ${AMQP_SERVER:+--amqp} ${AMQP_SERVER:-} || slumber
+    timeout 12h ./run-queue ${AMQP_SERVER:+--amqp} ${AMQP_SERVER:-} || sleep 10
```

Comment:
That means our AMQP server is now being hit every 0.25 seconds, due to the annoying non-blocking behaviour in `run-queue`. But really, we should fix that there instead.

Reply:
I agree.  I'll try to find the code I had to do that... from what I recall it's complicated by the fact that we need to wait on multiple queues at once...